files/zh-cn/web/api/web_audio_api/using_web_audio_api/index.html

```diff
@@ -167,7 +167,7 @@ <h2 id="修改声音">修改声音</h2>
 
 <p><img alt="an audio graph with an audio element source, connected to a gain node that modifies the audio source, and then going to the default destination" src="https://mdn.mozillademos.org/files/16196/graph2.jpg"></p>
 
-<p>默认增益为1；这使当前音量保持不变。增益可以设置的最小值约-3.4，最大约3.4。这里我们将允许音箱增益可以设置到2（2倍的原音量）和降低到0（这可以有效的静音）。</p>
+<p>默认增益为1；这使当前音量保持不变。增益可以设置的最小值约`-3.4028235E38`，最大约`3.4028235E38`。这里我们将允许音箱增益可以设置到2（2倍的原音量）和降低到0（这可以有效的静音）。</p>
```

Comment:
the inline code block will not be rendered in html, please use `<code>...</code>` instead.

suggestion:

```html
<p>默认增益为1；这使当前音量保持不变。增益可以设置的最小值约 <code>-3.4028235E38</code>，最大约 <code>3.4028235E38</code>。这里我们将允许音箱增益可以设置到2（2倍的原音量）和降低到0（这可以有效的静音）。</p>
```

Reply:
thanks for the suggestion